elements/lisk-p2p/src/p2p.ts

```diff
@@ -225,7 +242,10 @@ export class P2P extends EventEmitter {
 
 		this._handleOutboundPeerConnectAbort = (peerInfo: P2PPeerInfo) => {
 			const peerId = constructPeerIdFromPeerInfo(peerInfo);
-			if (this._triedPeers.has(peerId)) {
+			const isWhiteListed = this._peerListsWithoutConflicts.whitelisted.find(
+				peer => constructPeerIdFromPeerInfo(peer) === peerId,
+			);
+			if (this._triedPeers.has(peerId) && !isWhiteListed) {
```

Comment:
Shouldn't we also keep fixed peers in `triedPeers` list as we keep the connection with them or is it that we never save fixed peer in any of this list because of its importance and not to be advertised?

Reply:
We treat fixed peers separately because of the second reason. We should revisit the topic anyways in #3661